CircularProgressBar/script.js

```diff
@@ -4,6 +4,6 @@ const progress = document.getElementById("showProgress");
 
 slider.addEventListener("input", () => {
     const percentage = slider.value;
-    border.style.background = "conic-gradient(rgb(6, 214, 160) " + percentage + "deg, black 0deg)";
-    progress.innerHTML = slider.value + "%";
+    border.style.background = "conic-gradient(rgb(6, 214, 160) " + percentage + "%, black 0deg)";
```

Comment:
Eu usaria `%` nos dois, seguir um padrao
```suggestion
    border.style.background = "conic-gradient(rgb(6, 214, 160) " + percentage + "%, black 0%)";
```